src/app/route/MyAccountWishlist/MyAccountWishlist.style.scss

```diff
@@ -0,0 +1,77 @@
+.MyAccountWishlist {
+    --placeholder-animation: placeholderAnimation 2s linear infinite, appear 200ms forwards;
+    
+    &-Wrapper {
+        max-width: 1200px;
+        display: grid;
+        grid-template-columns: 20% 78%;
+        grid-column-gap: 2%;
```

Comment:
Let's have `grid-column-gap` constant, like `1/2rem` try using `fr` for columns (they automatically compensate for gaps)